docs/en/integrations/language-clients/java/index.md

```diff
@@ -5,126 +5,250 @@ keywords: [clickhouse, java, jdbc, client, integrate, r2dbc]
 
-| Format                  | Support            | Comment                                                               |
-| ----------------------- | ------------------ | --------------------------------------------------------------------- | --- |
-| AggregatedFunction      | :white_check_mark: | :warning: does not support `SELECT * FROM table ...`                  |
-| Array(\*)               | :white_check_mark: |                                                                       |
-| Bool                    | :white_check_mark: |                                                                       |
-| Date\*                  | :white_check_mark: |                                                                       |
-| DateTime\*              | :white_check_mark: |                                                                       |
-| Decimal\*               | :white_check_mark: | `SET output_format_decimal_trailing_zeros=1` in 21.9+ for consistency |
-| Enum\*                  | :white_check_mark: | can be treated as both string and integer                             |     |
-| Geo Types               | :white_check_mark: | Point, Ring, Polygon, and MultiPolygon                                |     |
-| Int\*, UInt\*           | :white_check_mark: | UInt64 is mapped to `long`                                            |     |
-| IPv\*                   | :white_check_mark: |                                                                       |
-| Map(\*)                 | :white_check_mark: |                                                                       |
-| Nested(\*)              | :white_check_mark: |                                                                       |
-| Object('JSON')          | :white_check_mark: |                                                                       |
-| SimpleAggregateFunction | :white_check_mark: |                                                                       |
-| \*String                | :white_check_mark: |                                                                       |
-| Tuple(\*)               | :white_check_mark: |                                                                       |
-| UUID                    | :white_check_mark: |                                                                       |
-
-### Driver API
-
-#### Connect to ClickHouse
-
-**URL Syntax**: `protocol://host[:port][/database][?param[=value][&param[=value]][#tag[,tag]]`, for example:
+### Configuration 
 
-- `http://localhost:8443?ssl=true&sslmode=NONE`
-- `http://(https://explorer@play.clickhouse.com:443`
-- `tcp://localhost?!auto_discovery#experimental),(grpc://localhost#experimental)?failover=3#test`
+All settings are defined by instance methods (a.k.a configuration methods) that make scope and context of each value clear. 
+Major configuration parameters are defined in one scope (client or operation) and do not override each other. Handling 
```

Comment:
Shouldn't `operation`-level parameter override a parameter defined on the instance level?

Reply:
Only in some cases, for example timeouts. 
We do not guarantee to make all settings overridable because in certain cases it would mean creating another client instance. For example, we may not turn-off connection pool for an operation.